nengo_viz/static/viz_netgraph.js

```diff
@@ -119,7 +120,7 @@ VIZ.NetGraph = function(parent, args) {
                     self.menu.show(event.clientX, event.clientY, 
                                    self.generate_menu());
                 }
-                event.stopPropagation();  
+                //event.stopPropagation();  
```

Comment:
> Why was this removed?

I was testing some different Tap events for the sliders and this was interferring. 

However I didn't end up using those tap events so I'll turn this back on.